SparkPost/2web2ui
# SA-827 Add Incident Details Page

### What Changed
 - Added an Incident Details Page which includes:
   A Panel with:
   - Date Listed
   - Date Resolved (or yellow active tag if not resolved)
   - Days Listed
   - List of up to 6 most recent historical incidents (same blacklist, same resource)
   - Button to link to reports/summary with the given parameters:
     - Date range of from 7 days before listing date to 7 days after resolved date (or current day if unresolved)
     - Filtered by sending domain or IP based on the incident's resource
  A Panel with:
   - List of up to 3 most recent incidents on _this_ incident's blacklist
   - If there are none, an empty state says so
  A Panel with:
   - List of up to 3 most recent incidents of _this_ incident's resource
   - If there are none, an empty state says so

### How To Test
 - Ensure all AC is there
 - Ensure loading states are good (each panel has separate loading state, since they are different endpoints

### To Do
- [ ] Add some more tests for code coverage
